aries-site/src/pages/components/footer.js

```diff
@@ -18,20 +45,162 @@ const Footer = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
-        <SubsectionText>
-          Footer is a Box with a set of preset properties. Box properties allow
-          you to customize the footer.
-        </SubsectionText>
+        <SubsectionText>{page.description}</SubsectionText>
         <Example
+          designer="https://designer.grommet.io/footer?id=HPE-design-system-hpedesignsystem-hpe-com"
           docs="https://v2.grommet.io/footer?theme=hpe"
-          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/layouts/FooterExample.js"
-          figma="https://www.figma.com/file/9aoCaf5lqzdQv1q4NFi0GN/hpe-design-system-library-footer"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/footer/FooterExample.js"
+          figma="https://www.figma.com/file/9aoCaf5lqzdQv1q4NFi0GN/HPE-Footer-Component?node-id=84%3A26"
           width="100%"
         >
           <FooterExample />
         </Example>
       </Subsection>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          Footer is used at the bottom of an application. It provides buttons
+          that link to company information and other important links to
+          secondary pages that may not be part of the{' '}
+          <Anchor label="Header" href={nameToPath('Header')} /> navigation
```

Comment:
header or footer ?

Reply:
In this case, header because we're saying certain things didn't exist in the Header so you may want to put them in footer.